README.md

```diff
@@ -1 +1,138 @@
-# First-Capstone-Project
+
+# First-Captone-Project
+
+# IT Audit
+
+This is a project acknowledging and affirming that I have properly understood everything learnt so far from the first module.
+
+> Learning objectives
+
+- I Used semantic HTML tags.
+- I Applied best practices in HTML code.
+- Used CSS selectors correctly.
+- Used CSS box model.
+- Used Flexbox to place elements on the page.
+- Demonstrated ability to create UIs adaptable to different screen sizes using media queries.
+- Used GitHub Pages to deploy web pages.
+- Applied JavaScript best practices and language style guides in code.
+- Used JavaScript to manipulate DOM elements.
+- Used JavaScript events.
+- Used objects to store and access data.
+- Communicated technical concepts to other technical people.
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+Click [here] [https://loom.com/share/f8bb0307b6ee457b8dc2c05882d193e5] to watch a walk-through description, given by me.
```

Comment:
I have watched both the loom and zoom videos. They both look nice and proffessional. :1st_place_medal:  Unfortunately, both are missing one key requirement for the video as shown below in the project documentation image (check the 3rd bullet point). Kindly make sure that you highlight some interesting piece of code or something you built that you are very proud of: :smiley: 


_NOTE: You only need to present one video. So either of them will be ok so far as all the project documentations is available_


![Screenshot from 2022-10-20 00-23-06](https://user-images.githubusercontent.com/65233860/196829920-8dc60a90-86c3-47b1-a18d-bec80de72b43.png)